administrator/components/com_languages/tmpl/language/edit.php

```diff
@@ -16,17 +16,23 @@
 
 JFactory::getDocument()->addScriptDeclaration(
 	'
-	jQuery(document).ready(function() {
-		jQuery("#jform_image").on("change", function() {
+	function changeImg(field, flag_id) {
+		jQuery("#" + field).on("change", function() {
 			var flag = this.value;
-			if (flag) {
-				jQuery("#flag img").attr("src", "' . JUri::root(true) . '" + "/media/mod_languages/images/" + flag + ".gif").attr("alt", flag);
+			if (flag && flag != "root") {
+				jQuery("#" + flag_id + " img").attr("src", "' . JUri::root(true)
+	. '" + "/media/mod_languages/images/" + flag + ".gif").attr("alt", flag);
 			}
 			else
 			{
-				jQuery("#flag img").removeAttr("src").removeAttr("alt");
+				jQuery("#" + flag_id + " img").removeAttr("src").removeAttr("alt");
 			}
-	});
+		});
+	}
+	
+	jQuery(document).ready(function() {
+		changeImg("jform_image", "flag");
+		changeImg("jform_params_fallback_lang", "fallback_lang_flag");
```

Comment:
Try avoiding jQuery, use vanilla JS (plain JS) -> http://youmightnotneedjquery.com this is very useful but any issue/question you have just write me :)